src/main/java/com/taskviewer/api/web/rq/RqTaskUpdate.java

```diff
@@ -0,0 +1,94 @@
+package com.taskviewer.api.web.rq;
+
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+
+public record RqTaskUpdate(
+  String title,
+  String about,
+  String username,
+  String status,
+  Integer priority,
+  LocalDateTime estimate,
+  LocalDateTime tracked
+  ) {
+
+  @Contract(value = " -> new", pure = true)
+  public static @NotNull RqTaskUpdate.TaskUpdateSqlBuilder taskUpdateSqlBuilder() {
+    return new RqTaskUpdate.TaskUpdateSqlBuilder();
+  }
+
+  public static class TaskUpdateSqlBuilder {
+    private final StringBuilder sql = new StringBuilder("UPDATE task set updated = now()");
+
+    public TaskUpdateSqlBuilder withTitle(String title) {
+      if (title != null) {
+        sql.append(", title = '")
+          .append(title)
+          .append("'");
+      }
+      return this;
+    }
+
+    public TaskUpdateSqlBuilder withAbout(String about) {
+      if (about != null) {
+        sql.append(", about = '")
+          .append(about)
+          .append("'");
+      }
+      return this;
+    }
+
+    public TaskUpdateSqlBuilder withAssigne(String username) {
```

Comment:
assignee

Reply:
This way, you also have grammar error in liquibase file, where «estimate».

Reply:
@h1alexbel fixes grammar error